### DISCUSSION TITLE
Using PTZ on Blueiris for Yi Dome Guard YRS r35gb

### DISCUSSION THREAD

**Discussion by onisahkku** [roleoroleo/yi-hack-Allwinner-v2#743] [Q&A] [answered]
Hi  !
I've bought and hacked my new Yi Dome Guard.
Everything's working good, but i can't find how to(if possible) to use PTZ on Blueiris...

Could someone try to help me ? :)

Edit : After looking everywhere, i've found this :
https://github.com/roleoroleo/yi-hack-Allwinner-v2/wiki/Web-services-description#ptz
A beginning of solution. But i've to push multiple times to move my camera from one side to the other. Any ideas to have a "i let right button pushed = it keeps to move right ?"

**Comment by onisahkku** [comment]
There is a CUSTOM HTTP option. Would it be possible to send PTZ actions via HTTP ?
![Capture d'écran 2023-11-08 111003](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/3154897/cab59e86-87b0-4776-886e-9abe024c1079)

**Reply by roleoroleo** [reply]
There is no method to start a movement without stop it in the same command.
But you can use a very high time period.
i.e.
`http://ip/cgi-bin/ptz.sh?dir=right&time=10`

**Reply by roleoroleo** [reply]
Otherwise onvif supports continuous movements.

**Reply by onisahkku** [reply]
Blue Iris can't detect ONVIF. What port is used for ONVIF ? 8999 and 9000 seems not working... :(

**Reply by roleoroleo** [reply, marked as answer]
Onvif is on port 80, it shares the port with the web gui.
http://IP_CAM/onvif/device_service

**Reply by onisahkku** [reply]
![Capture d'écran 2023-11-12 110922](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/3154897/fbcabb5d-88d9-480b-a97c-6518760a08ab)
![Capture d'écran 2023-11-12 1110062](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/3154897/dcc16d0f-93ba-4bf5-8f4f-40756da5e96c)
Thanks ! I join my blueiris parameters if someone intersted :)

**Reply by tummerry** [reply]
what APP

**Comment by onisahkku** [comment]
> what APP

Blue Iris (first Google result)
